src/Compositor.cpp

```diff
@@ -884,6 +884,20 @@ void CCompositor::moveWindowToTop(CWindow* pWindow) {
     }
 }
 
+void CCompositor::moveWindowToWorkspace(CWindow* pWindow, int workspaceID) {
```

Comment:
can you move it to CWindow plox cutie

Reply:
@vaxerski done

Btw, seems like there is already a `CCompositor::moveWindowToWorkspace` but with different signature. I couldn't find any usages of it.

I forgot this language had overloading xD

Reply:
lul thanks removed it